.github/workflows/build-check-install.yaml

```
@@ -176,6 +194,22 @@ jobs:
               writeLines(readLines(check_log))
               stop("⚠ R CMD check has warnings")
             }
+            if ("${{ inputs.check-note-blocklist }}" == "true") {
+              print("Checking notes...")
+              regexes <- "${{ inputs.note-blocklist }}"
+              regexes <- unlist(strsplit(regexes, split = "\n"))
+              lines = paste(readLines(check_log), collapse = " ")
```

keep assignment operator consistent throughout your script
```suggestion
              lines <- paste(readLines(check_log), collapse = " ")
```